client/asset/dcr/dcr.go

```diff
@@ -1620,7 +1760,7 @@ func (dcr *ExchangeWallet) sendWithReturn(baseTx *wire.MsgTx, addr dcrutil.Addre
 	// admittedly un-scientific. This should account for any signature length
 	// related variation as well as a potential dust change output with no
 	// subtractee specified, in which case the dust goes to the miner.
-	if checkRate > feeRate*3 {
+	if changeAdded && checkRate > feeRate*3 {
```

Comment:
With the changes in `TestAvailableFund`, I actually ran up against the `*3` when the change was a dust ouptut. `*3` was `69` atoms/byte and the effective rate ended up being `71`. I'm sure we could improve this still.